upgrade/1.0/scripts/upgrade/ncn-upgrade-ceph-initial.sh

```diff
@@ -2,10 +2,6 @@
 #
 # Copyright 2021 Hewlett Packard Enterprise Development LP
 #
-set -e
```

Comment:
Adding this back in per Craig's comment.

Reply:
done